tcms/testruns/templates/testruns/get.html

```diff
@@ -16,6 +16,7 @@ <h1 class="col-md-12" style="margin-top: 0">
             data-pk="{{ object.pk }}"
             data-perm-remove-tag="{{ perms.testruns.delete_testruntag }}"
             data-trans-no-executions-selected="{% trans 'No executions selected! Please select at least one execution!'%}"
+            data-trans-enter-assignee-name-or-email="{% trans 'Enter username, email or ID of the new assignee:'%}"
```

Comment:
Let's use "Enter username, email or user ID" as a generic message that can be used anywhere where these 3 values are accepted. Will minimize source strings.